db/identifiers.py

```diff
@@ -30,9 +33,12 @@ def truncate_if_necessary(identifier):
 
 
 def is_identifier_too_long(identifier):
-    postgres_identifier_size_limit = 63
+    # TODO we should support POSTGRES_IDENTIFIER_SIZE_LIMIT here;
+    # This is our current limit due to an unknown bug that manifests at least
+    # when importing CSVs.
+    our_temporary_identifier_size_limit = 57
```

Comment:
Brent had set this to 48, but I don't know why. I suggest we consider that that was a mistake or an offhand choice. The added test passes as long as the identifier is no longer than 57 bytes, so that's what I set the limit to.

Through the frontend, I did an import of a CSV file with 63+ byte long column names, shuffled the columns about a bit, and didn't get any errors.

Reply:
I didn't get any error when I tried importing a CSV file with a long name. @mathemancer Is there a reason behind choosing 48 bytes as the size limit when we the max limit without causing any error is 59 bytes?

Reply:
@mathemancer mentioned that he _wanted to leave a buffer in case the truncation went further for some unknown reason_ @dmos62 Can we change the `identifier_size_limit` to `48` just to be safe?

Reply:
Sure.